app/middlewares/auth.js

```diff
@@ -23,3 +23,11 @@ exports.isAuthenticated = (req, res, next) => {
     next(errors.defaultError('User is not logged in'));
   }
 };
+
+exports.isAdmin = (req, res, next) => {
+  if (req.user.isAdmin) {
+    next();
+  } else {
+    next(errors.defaultError('User is not allowed'));
```

Comment:
Same as before, we should fail with a more specific error so it translates to a more descriptive http status code.